ml_core/extractors/identifiers.py

```diff
@@ -0,0 +1,20 @@
+# TODO (Guillemdb): fix imports
```

Comment:
Looks like they are fixed, aren't they?
Here are everywhere

Reply:
Let me fix all of this in the next PR. I already corrected most of the minor style improvements I could find in a different branch. Once I fix everything which is not code related I will push it.

Reply:
can you explain what is wrong with these imports?

Reply:
Nothing, I forgot to remove the TODO.